cloudburst/client/client.py

```diff
@@ -299,11 +305,22 @@ def exec_func(self, name, args):
         return r.response_id
 
     def _connect(self):
+        logging.info('Connecting to scheduler service')
```

Comment:
Probably unnecessary logging given we log if we retry.

Reply:
removed